Doc/reference/datamodel.rst

```diff
@@ -1946,10 +1945,11 @@ Once the appropriate metaclass has been identified, then the class namespace
 is prepared. If the metaclass has a ``__prepare__`` attribute, it is called
 as ``namespace = metaclass.__prepare__(name, bases, **kwds)`` (where the
 additional keyword arguments, if any, come from the class definition). The
-``__prepare__`` method should be implemented as a :func:`classmethod`.
```

Comment:
We lost a line with valuable info here. Please keep it and add your changes after.